keel-core/src/main/kotlin/com/netflix/spinnaker/keel/activation/DiscoveryActivator.kt

```diff
@@ -1,13 +1,13 @@
-package com.netflix.spinnaker.keel.eureka
+package com.netflix.spinnaker.keel.activation
 
-import com.netflix.appinfo.InstanceInfo.InstanceStatus.UP
-import com.netflix.spinnaker.keel.activation.ApplicationDown
-import com.netflix.spinnaker.keel.activation.ApplicationUp
-import com.netflix.spinnaker.kork.eureka.RemoteStatusChangedEvent
+import com.netflix.spinnaker.kork.discovery.InstanceStatus.UP
+import com.netflix.spinnaker.kork.discovery.RemoteStatusChangedEvent
 import org.slf4j.LoggerFactory
 import org.springframework.context.ApplicationEventPublisher
 import org.springframework.context.ApplicationListener
+import org.springframework.stereotype.Component
 
+@Component
 class DiscoveryActivator(
```

Comment:
How does this continue to work? What happens when running locally and there's no Eureka?

Reply:
https://github.com/spinnaker/kork/blob/master/kork-core/src/main/java/com/netflix/spinnaker/kork/discovery/NoDiscoveryStatusPublisher.java

Reply:
All of the boilerplate code that does discovery stuff that used to be in services was promoted into kork.

Reply:
Oh sweet. I think we just created our own event types to decouple things from directly depending on Eureka. I'll see if I can get rid of this class.